larpandoracontent/LArMonitoring/MuonLeadingEventValidationAlgorithm.cc

```diff
@@ -269,11 +269,14 @@ void MuonLeadingEventValidationAlgorithm::ProcessOutput(
             continue;
 
         // Cosmic ray parameters
-        int ID_CR, nReconstructableChildCRLs(0);
+        int nReconstructableChildCRLs(0), nCorrectChildCRLs(0);
+
+#ifdef MONITORING
+        int ID_CR;
```

Comment:
Completely separately to the aims of this PR, would suggest that these variables are initialised with some defined values, anything really is better than being undefined.